helpers/repos.py

```diff
@@ -0,0 +1,92 @@
+
+    def __eq__(self, other):
+        return self.gh_id == other.gh_id
+
+    def __hash__(self):
+        return hash(self.gh_id)
+
+
+# To get the GitHub ID of a repo, see <https://stackoverflow.com/a/47223479/393146>.
+
+HALO2_REPOS = [
+    Repo(('zcash', 'halo2'), 290019239, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMyNzg1NDUx'),
+    Repo(('zcash', 'pasta_curves'), 344239327, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMyOTI3Njg2'),
+]
+
+CORE_REPOS = [
+    Repo(('zcash', 'zcash'), 26987049, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvOTc3ODc2NQ'),
+    Repo(('zcash', 'zips'), 47279130, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMjIwMzQwMDY'),
+    Repo(('zcash', 'incrementalmerkletree'), 48303644, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzMDcwMDc5'),
+    Repo(('zcash', 'librustzcash'), 85334928, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTg5MDU1NTE'),
+    Repo(('zcash', 'zcash-test-vectors'), 133857578, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMyOTMxNTEx'),
+    Repo(('zcash', 'sapling-crypto'), 111058300, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzOTY3ODY4'),
+    Repo(('zcash', 'orchard'), 305835578, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMyODU2MzA2'),
+    Repo(('zcash', 'zip32'), 141066493, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzOTY2MzAy'),
+] + HALO2_REPOS
+
+TFL_REPOS = [
+    Repo(('Electric-Coin-Company', 'tfl-book'), 642135348, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzOTE2NDQz'),
+    Repo(('Electric-Coin-Company', 'zebra-tfl'), 725179873, None),
+    Repo(('Electric-Coin-Company', 'simtfl'), 695805989, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzOTQ0MTk5'),
+]
+
+ANDROID_REPOS = [
+    Repo(('Electric-Coin-Company', 'zashi-android'), 390808594, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzMDEwMTMw'),
+    Repo(('Electric-Coin-Company', 'zcash-android-wallet-sdk'), 151763639, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTg5MDI4MjE'),
+    Repo(('Electric-Coin-Company', 'zashi'), 719178328, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzOTYwOTgw'),
+]
+
+IOS_REPOS = [
+    Repo(('Electric-Coin-Company', 'zashi-ios'), 387551125, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzMDEwMTI5'),
+    Repo(('Electric-Coin-Company', 'zcash-swift-wallet-sdk'), 185480114, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTg5MDU1NjE'),
+    Repo(('Electric-Coin-Company', 'MnemonicSwift'), 270825987, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMyNzk3Mzg0'),
+    Repo(('Electric-Coin-Company', 'zcash-light-client-ffi'), 439137887, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzMTMwNjcy'),
+    Repo(('Electric-Coin-Company', 'zashi'), 719178328, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzOTYwOTgw'),
+]
+
+WALLET_REPOS = [
+    Repo(('zcash', 'librustzcash'), 85334928, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTg5MDU1NTE'),
+    Repo(('zcash', 'lightwalletd'), 159714694, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTg5MDU1NzE'),
+] + ANDROID_REPOS + IOS_REPOS
+
+ECC_REPOS = CORE_REPOS + TFL_REPOS + WALLET_REPOS + [
+    Repo(('Electric-Coin-Company', 'infrastructure'), 65419597, 'Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMjIwNTA1NjY'),
+]
+
+ZF_REPOS = [
+    Repo(('ZcashFoundation', 'zebra'), 205255683, None),
```

Comment:
Turns out that the ZenHub GraphQL API key doesn't actually let us enumerate repos, so we'd need ZF to run the new script I added to get those ZenHub IDs.